packages/web/src/components/withdraw-usdc-modal/WithdrawUSDCModal.tsx

```diff
@@ -10,24 +10,26 @@ import {
   formatUSDCWeiToFloorCentsNumber,
   Nullable,
   withdrawUSDCSelectors,
-  Status
+  Status,
+  Name
 } from '@audius/common'
 import { Modal, ModalContent, ModalHeader } from '@audius/stems'
 import BN from 'bn.js'
-import { Formik, FormikProps } from 'formik'
+import { Formik, FormikProps, useFormikContext } from 'formik'
 import { useDispatch, useSelector } from 'react-redux'
 import { z } from 'zod'
 import { toFormikValidationSchema } from 'zod-formik-adapter'
 
 import { ReactComponent as IconTransaction } from 'assets/img/iconTransaction.svg'
 import { Icon } from 'components/Icon'
 import { Text } from 'components/typography'
+import { make, track } from 'services/analytics'
 import { isValidSolAddress } from 'services/solana/solana'
 
 import styles from './WithdrawUSDCModal.module.css'
 import { ConfirmTransferDetails } from './components/ConfirmTransferDetails'
 import { EnterTransferDetails } from './components/EnterTransferDetails'
-import { Error } from './components/Error'
+import { ErrorPage } from './components/ErrorPage'
```

Comment:
I originally renamed this component because I was trying to send a proper `Error` in the events and this was shadowing the built-in class such that I couldn't do `new Error()`. I ended up sending just a string of the error message instead. But I figured this was good to leave in so that we're not shadowing :-)